OnlinerTask/OnlinerTask.BLL/Services/ElasticSearch/ProductLogger/IProductLogger.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading.Tasks;
+using OnlinerTask.Data.SearchModels;
+
+namespace OnlinerTask.BLL.Services.ElasticSearch.ProductLogger
+{
+    public interface IProductLogger
+    {
+        Task LogAdding(int productId, ProductModel productModel);
+
+        Task RemoveLog(int productId);
```

Comment:
I think, it`s unnecessary to have log of product, which has been unchecked(and in this case we will always have logs of products, that interesting to users)